activities.py

```diff
@@ -239,11 +239,11 @@ def __init__(self, spec_url):
         self.orig_url = spec_url
         self.data = {
             "id": u"",
-            "title": None,
+            "title": "",
             "description": None,
             "ciuName": None,
-            "org": None,
-            "url": None,
+            "org": "Proposal",
```

Comment:
How is org "Proposal"? Shouldn't it be the place where proposal has been made? Or perhaps this default value could be
"WhatWG|W3C|IETF|etc" to hint what the value should be?